### ISSUE TITLE
Feature proposal: switch to another database system ?

### ISSUE THREAD

**Issue by lapineige** [YunoHost-Apps/garage_ynh#8]
As performance might be critical for such a system, maybe it would be good to switch to another database than mysql, as recommended by Garage documentation ? https://garagehq.deuxfleurs.fr/documentation/reference-manual/configuration/#db-engine-since-v0-8-0

What do you think ?

And how can we convert this on Yunohost side ? For both fresh installs and upgrades.

**Comment by oiseauroch** [comment]
I agree but last time i tried to do so, if get the following error on raspi4 : https://git.deuxfleurs.fr/Deuxfleurs/garage/issues/439. The issue is closed but there is still no solution and I don't want to support several database system for now, unfortunatly I don't have time to do so.

**Comment by lapineige** [comment]
Thanks for your feedback :)
I'm closing then.

**Comment by lapineige** [comment]
I'm reopening : since Garage 0.9 the old database systems are deprecated.
We should use LMDB by default, and migrate the old installation.

**Comment by mdornseif** [comment]
K think te current version does not use mysql anymore and this ticket van be closed. See also https://github.com/YunoHost-Apps/garage_ynh/issues/47

**Comment by Gredin67** [comment]
both sqlite and lmdb can now be installed, depending on the use-case garage cluster is destined to.
